src/pymor/reductors/ph/basic.py

```diff
@@ -0,0 +1,67 @@
+from pymor.algorithms.projection import project, project_to_subbasis
+from pymor.models.iosys import PHLTIModel
+from pymor.reductors.basic import ProjectionBasedReductor
+
+
+class PHLTIPGReductor(ProjectionBasedReductor):
+    """Petrov-Galerkin projection of an |PHLTIModel|.
+
+    Parameters
+    ----------
+    fom
+        The full order |PHLTIModel| to reduce.
+    V
+        The basis of the ansatz space.
```

Comment:
```suggestion
        The basis of the trail space.
```

Reply:
I guess you wanted to suggest 'trial', @pmli ...

Reply:
Indeed...